charts/aws-vpc-cni/crds/customresourcedefinition.yaml

```diff
@@ -1,10 +1,9 @@
-{{- if .Values.crd.create -}}
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: eniconfigs.crd.k8s.amazonaws.com
   labels:
-{{ include "aws-vpc-cni.labels" . | indent 4 }}
+    k8s-app: aws-node
```

Comment:
any reason to add this label?

Reply:
I thought we are using this label, seems like not. I'll be removing this and keep it consistent with the crd that gets installed by default on a cluster